src/App.scss

```diff
@@ -135,14 +119,15 @@ body {
     border-radius: 3px;
     margin-right: 0.5rem;
     margin-top: 0.25rem;
-    grid-template-rows: 1fr min-content;
+    grid-template-rows: 1fr min-content min-content;
     background-color: #CCE6E2;
 }
 
 .App__world-selector {
     display: flex;
     flex-direction: column;
     margin-top: 1rem;
+    border-bottom: 1px solid #88C3BC;
```

Comment:
We need to update this border colour in different themes.

Dark Theme: #33717F
Grayscale Theme: #9DA4AF
Contrast Theme: #FFFFFF

Reply:
Good catch, and fixed.